Packs/Jira/Integrations/JiraV3/JiraV3.yml

```diff
@@ -1582,6 +1582,15 @@ script:
       name: lastUpdate
     description: Get remote data from a remote incident. This method does not update the current incident, and should be used for debugging purposes only.
     name: get-remote-data
+  - arguments:
+      - description: The remote incident ID.
+        name: id
+        required: true
+      - defaultValue: '0'
+        description: The UTC timestamp in seconds of the last update. The incident is only updated if it was modified after the last update time.
+        name: lastUpdate
```

Comment:
is last update even relevant in this case?

Reply:
I don't believe so. I'm not sure how they would even populate it. I'll pull it.

Reply:
Just made a large change which addresses this